bin/alluxio

```diff
@@ -25,6 +25,8 @@ function printUsage {
   echo -e "  runTests      \t Run all end-to-end tests on an Alluxio cluster."
   echo -e "  runJournalCrashTest\t Test the Master Journal System in a crash scenario. Try 'alluxio runJournalCrashTest -help' for more help."
   echo -e "                \t NOTE: This command will stop the existing server and creates a new one!"
+  echo -e "  runMesosTest  \t Test the Alluxio integration with Mesos. Try 'alluxio mesosTest -help' for more help."
+  echo -e "                \t NOTE: This command requires Mesos to be running and will will stop the existing Alluxio server"
```

Comment:
`will will stop the existing Alluxio server` -> `will stop any Alluxio servers that are currently running`

This command doesn't require Alluxio to be running right?